store/store.go

```diff
@@ -659,3 +659,50 @@ func (s *PebbleStore) GetProcessedTickIntervals(ctx context.Context) ([]*protobu
 
 	return processedTickIntervals, nil
 }
+
+func (s *PebbleStore) SetEmptyTicksPerEpoch(epoch uint32, emptyTicksCount uint32) error {
+	key := emptyTicksPerEpochKey(epoch)
+
+	value := make([]byte, 4)
+	binary.LittleEndian.PutUint32(value, emptyTicksCount)
+
+	err := s.db.Set(key, value, pebble.Sync)
+	if err != nil {
+		return errors.Wrapf(err, "saving emptyTickCount for epoch %d", epoch)
+	}
+	return nil
+}
+
+func (s *PebbleStore) GetEmptyTicksPerEpoch(epoch uint32) (uint32, error) {
+	key := emptyTicksPerEpochKey(epoch)
+
+	value, closer, err := s.db.Get(key)
+	if err != nil {
+		if errors.Is(err, pebble.ErrNotFound) {
+			return 0, nil
+		}
+
+		return 0, errors.Wrapf(err, "getting emptyTickCount for epoch %d", epoch)
+	}
+	defer closer.Close()
+
+	emptyTicksCount := binary.LittleEndian.Uint32(value)
+
+	return emptyTicksCount, nil
+}
+
+func (s *PebbleStore) GetEmptyTicksForAllEpochs(epochs map[uint32]uint32) (map[uint32]uint32, error) {
```

Comment:
why do you need a map as the input if you only use the keys?

why is the method called GetEmptyTicksForAllEpochs if you need to specify the epochs as a param?